lib/app/hooks/configure.go

```diff
@@ -375,13 +375,13 @@ TMPDIR={{.StateDir}} {{.StateDir}}/gravity --state-dir={{.StateDir}} app unpack
 	--insecure --ops-url=$ops_url \
 	{{.Package}} {{.ResourcesDir}};
 mv {{.ResourcesDir}}/resources/* {{.ResourcesDir}}
```

Comment:
Agree with @r0mant here.

I think we should fix the `app unpack` command to extract the contents of `<package>/resources/` and not `<package>/.` as it does now. Since this is the only user of `app unpack`, it is relatively safe to change the semantics of the command, although we can always add a new hidden flag to support this mode to make it backwards-compatible if required.
The main issue is moving sub-directory called the same as the existing directory in the target directory (`resources` in this case), so I'd simply try and eliminate the need for mv in the first place - i.e. extract the contents of `./resources` directory without the directory itself.